docs/user-tutorial/result-summary.md

```diff
@@ -0,0 +1,122 @@
+---
+id: result-summary
+---
+
+# Result Summary
+
+## Introduction
+
+This tool is to generate a readable summary report based on the raw benchmark results of single or multiple machines.
+
+## Usage
+
+1. [Install SuperBench](../getting-started/installation) on the local machine.
+
+2. Prepare the raw data, and rule file under the current path or somewhere on the local machine.
+
+3. After installing the Superbnech and the files are ready, you can start to generate the result summary automatically using  `sb result summary` command. The detailed command can be found from [SuperBench CLI](../cli).
```

Comment:
`3. Generate the result summary automatically using `sb result summary` command. The detailed command can be found from [SuperBench CLI](../cli). `

Reply:
one more space after `using`. Please remove it.

Reply:
updated